sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Helpers.cs

```diff
@@ -52,10 +52,15 @@ internal static EventDefinition GetEventDefintionFromPayload(string payload)
 
         internal static HttpResponseMessage HttpErrorResponse(Exception ex)
         {
-            return new HttpResponseMessage(System.Net.HttpStatusCode.BadRequest)
+            var response = new HttpResponseMessage(System.Net.HttpStatusCode.BadRequest)
             {
                 Content = new StringContent(GetFailedRequestPayload(ex))
             };
+
+            // Set the metrics on header
+            EventTriggerMetrics.SetMetricHeaders(response);
```

Comment:
This is an unusual thing to do and not a common pattern for Function extensions.  What's the rationale behind this, and has it been discussed with the Functions team to ensure that it doesn't go against platform conventions?   

This is another area that should be discussed with the the .NET architects as well.

//cc: @KrzysztofCwalina, @tg-msft, @fabiocav

Reply:
I spoke with our identity libraries team on metrics and they even helped decide which metrics we should be collecting. Then ones we are collecting are a subset of theirs for now. Have not spoken with with functions extension team. Will start a conversation now.

Reply:
@KrzysztofCwalina modified the metrics to match the general guidelines for azure sdk.